### PR TITLE
Conditional Region Support

### PR DESCRIPTION
**Pull request by phschaad**
Make sure the extension works with Conditoinal regions and Loop regions, to the extent they will be present / used in DaCe 1.0.

Relies on https://github.com/spcl/dace-webclient/pull/180.

### CODE REVIEW

**Comment by tbennun** [comment]
Could we run that again? @phschaad

File: tsconfig.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
 	"compilerOptions": {
 		"module": "commonjs",
-		"target": "ES2022",
+		"target": "ESNext",
```
Comment (tbennun) [inline tsconfig.json:4]: oh dear why. Which feature necessitates this and is there good support in Electron?

**Review by tbennun — APPROVED** [review]
See question

File: tsconfig.json (hunk 1 above)
Comment (phschaad) [reply tsconfig.json]: Very good catch, I don't recall why I modified this. I have reverted it to ES2022, which is also what the webclient now depends on following this PR: https://github.com/spcl/dace-webclient/pull/181

Note that we _would_ in theory be fine with `ESNext`, as VSCode's version of Electron uses a version of the V8 engine which supports quite a few features that are not even in the most recent ECMA standard yet, but this is definitely the safer option.

**Pull request merged**